src/main/scala/cromwell/engine/backend/BackendType.java

```diff
@@ -1,5 +1,16 @@
 package cromwell.engine.backend;
 
 public enum BackendType {
-    LOCAL, JES, SGE;
+    LOCAL {
+        @Override
+        public String displayName() {
+            return "Local";
+        }
+    },
+    JES,
+    SGE;
```

Comment:
We do not need or want an enum for pluggable backends, this is a vestige of compiled-in backends that we're not ready to get rid of yet.